velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -66,105 +80,85 @@ class NthValueTest : public WindowTestBase {
     VectorFuzzer fuzzer(options, pool_.get(), 0);
     return fuzzer.fuzz(type);
   }
+
+  const std::string frameClause_;
+};
+
+class MultiFrameNthValueTest : public NthValueTest,
```

Comment:
What is the difference between MultiFrameNthValueTest and MultiNthValueTest ? They look identical.